pkg/sql/lex/BUILD.bazel

```diff
@@ -31,3 +31,44 @@ go_test(
         "//pkg/sql/sessiondatapb",
     ],
 )
+
+go_binary(
+    name = "all_keywords",
+    srcs = ["all_keywords.go"],
+    gotags = ["all-keywords"],
+)
+
+# Define the target to auto-generate pkg/sql/lex/keywords.go from the grammar file.
+genrule(
+    name = "keywords_go",
+    srcs = [
+        "//pkg/sql/parser:sql.y",
+    ],
+    outs = ["keywords.go"],
+    cmd = """
+        $(location :all_keywords) < $< > $@.tmp || rm $@.tmp
```

Comment:
I think you can remove the `.tmp` stuff, so it would look like:

```
$(location :all_keywords) < $< > $@
gofmt -s -w $@
```

Also make the same change in all the other `genrule`s accordingly.

Bazel isn't as dumb as `make`, so this shouldn't be necessary :)